pkg/sentry/socket/hostinet/socket_unsafe.go

```diff
@@ -90,26 +90,50 @@ func getsockopt(fd int, level, name int, optlen int) ([]byte, error) {
 	return opt[:optlen32], nil
 }
 
+// GetAddress reads an sockaddr struct from the given address and converts
+// it to the format of SockAddrInet or SockAddrInet6 or raw bytes depending
+// on family.
```

Comment:
GetAddress reads an sockaddr struct as bytes and converts it to a SockAddrInet or SockAddrInet6 if it is an IPv4 or IPv6 address.